cmake/projects/Boost/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -315,17 +315,21 @@ if(NOT python_version STREQUAL "")
   if(NOT EXISTS "${python_include_directory}")
     hunter_internal_error("Directory not found: ${python_include_directory}")
   endif()
-  execute_process(
-    COMMAND "${python_executable}" -c "import os.path, sysconfig; info = sysconfig.get_paths(); print(os.path.dirname(info['stdlib']))"
-    RESULT_VARIABLE python_process
-    OUTPUT_VARIABLE python_library_directory
-    OUTPUT_STRIP_TRAILING_WHITESPACE
-  )
-  if(NOT python_process EQUAL 0)
+  find_package(PythonLibs ${python_version} EXACT QUIET)
+  if(NOT PYTHONLIBS_FOUND)
     hunter_user_error("Python runtime library directory for Python version ${python_version} not found.")
   endif()
-  if(NOT EXISTS "${python_library_directory}")
-    hunter_internal_error("Directory not found: ${python_library_directory}")
+  list(LENGTH PYTHON_LIBRARIES python_libraries_list_length)
+  if(python_libraries_list_length EQUAL 1)
+    set(python_optimized_library_path "${PYTHON_LIBRARIES}")
+  else()
+    list(GET PYTHON_LIBRARIES 1 python_optimized_library_path)
+  endif()
+  get_filename_component(python_library_directory "${python_optimized_library_path}" DIRECTORY)
+  if("@MSVC@")
```

Comment:
What toolchain you have used for Visual Studio testing? `Boost-python` example is not working for me with `vs-14-2015-win64-sdk-8-1` toolchain, error is:
```
boost/python/detail/wrap_python.hpp(50): fatal error C1083: Cannot open include file: 'pyconfig.h': No such file or directory
```

Compiler flags with include don't have slashes at all:
```
"-IC:softdeveloppythonx643.4.2Include"
```
I guess it should be `C:\soft\develop\python\x64\3.4.2\include`. That is the directory where `pyconfig.h` is located.

Reply:
Well, that's a bummer. You're right, it's not finding the includes because the path is missing backslashes.
It should be:
`C:\\soft\\develop\\python\\x64\\3.4.2\\include`

I've only tested with the default toolchain for msvc 15/2017.

I have an idea for a fix, though. FindPythonLibs can be used to retrieve the include directory as well, instead of manually querying sysconfig through the python interpreter. Hopefully, that will generate the correct path. I'll get back to you with an update as soon as I can.

Reply:
Will make new PR